configs/mixer/TemplateExtensions.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package istio.mixer.v1.config.template;
+import "google/protobuf/descriptor.proto";
+
+enum TemplateVariety {
+  TEMPLATE_VARIETY_UNSPECIFIED = 0;
+  TEMPLATE_VARIETY_CHECK = 1;
+  TEMPLATE_VARIETY_REPORT = 2;
+}
+
+extend google.protobuf.FileOptions {
```

Comment:
Surprisingly I was able to compile all the templates... and it worked.. may be proto3 has started supporting extensions.